test-cases/tests/alerts/c08-verify-alert-links-to-sops.md

```diff
@@ -0,0 +1,30 @@
+---
+targets:
+  - 2.3.0
+---
+
+# C08 - Verify alert links to SOPs
+
+## Description
+
+This test case should verify that all SOP links in alerts point to correct SOPs, that links to scaling_plans were removed from alerts and that scaling plans are moved to SOPs or referenced from SOPs.
+
+More info: <https://issues.redhat.com/browse/INTLY-7745>
```

Comment:
```suggestion
More info: https://issues.redhat.com/browse/INTLY-7745
```
It's a small thing, but maybe we don't need the `<`

Reply:
Tried different tools to check MD syntax and they aways complain about URLs not being inside `<...>`. Also when preview is rendered these angle brackets won't be visible. Would it be ok to leave them then?

Reply:
Didn't know, if that's the case than is perfectly fine for me :+1: 

Just another small stuff, what do you think of using `Requirement(s):` instead of `More info:` :thinking:

Reply:
`More info:` IMO better describes what the link is for - additional context for the test case, why it was created and so on.

Reply:
Is a common process to connect a Test Case to one or more Requirements, where a Requirement is a product future or functionality (From [Wikipedia](https://en.wikipedia.org/wiki/Requirement): `a requirement is a singular documented physical or functional need that a particular design, product or process aims to satisfy`), which means we could see Jira Tasks/Bugs/Epics as product Requirements.

So by using the `Requirement(s):` I would like to say that "the Jira listed are the product requirements that the test case aims to cover"

But I see that if it could be confused with "the Jira listed are required from the test case before the test can be executed"

I'm fine with using `More info` but I would also like to start computing the test coverage 
in the future (percentage of Requirements covered in Test Cases), therefore I think that starting to collect the Requirements covered in each test case could be a good start

Reply:
yes, makes sense